Devs-AA/Barefoot
# #168662048 Builds sign in with Facebook and google

#### What does this PR do?
IT signs user in with facebook and google
#### Description of Task to be completed?
None
#### How should this be manually tested?
1. ```git clone https://github.com/Devs-AA/Barefoot.git```  and checkout into ft-logout-168662046 branch
2. run `npm install` and create  a `.env file` with the key value pair as specified on the `.env.example` file
3. run `npm run migrate:dev` and `npm run seed:dev`

4. Visit the route `http://localhost:3300/api/v1/users/auth/token/google?access_token=${process.env.access_token_google}` where access_token is the user token which is generated from the client side and sent to the backend. If it is correct the user should be logged in and data saved in the database. Use the access_token from the `.env.example`

REPEAT THE SAME PROCEDURE FOR THE ROUTE
`http://localhost:3300/api/v1/users/auth/token/google?access_token=${process.env.access_token_facebook}` but this time around with an env value of access_token_facebook

None
#### What are the relevant pivotal tracker stories?
[#168662048](https://www.pivotaltracker.com/story/show/#168662046)

#### Screenshots (if appropriate)

![image](https://user-images.githubusercontent.com/38135488/65823990-54d8bb00-e258-11e9-8e48-0c6598e093a6.png)

Test coverage:

![image](https://user-images.githubusercontent.com/38135488/65823976-3377cf00-e258-11e9-8252-67e125e8dd91.png)


#### Questions:
